flink/v1.15/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSink.java

```diff
@@ -197,15 +197,13 @@ public void testJobHashDistributionMode() {
         .updateProperties()
         .set(TableProperties.WRITE_DISTRIBUTION_MODE, DistributionMode.HASH.modeName())
         .commit();
-
-    AssertHelpers.assertThrows(
-        "Does not support range distribution-mode now.",
-        IllegalArgumentException.class,
-        "Flink does not support 'range' write distribution mode now.",
-        () -> {
-          testWriteRow(null, DistributionMode.RANGE);
-          return null;
-        });
+    Assertions.assertThatThrownBy(
+            () -> {
+              testWriteRow(null, DistributionMode.RANGE);
+            })
+        .as("Does not support range distribution-mode now.")
```

Comment:
not needed as it's implicit through the expected error msg. The same is true for most other places